File: src/main/java/guru/xgm/language/lni/psi/LniTokenSets.java
package guru.xgm.language.lni.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.language.lni.psi.LniTypes.*;

public interface LniTokenSets {
    TokenSet COMMENTS = TokenSet.create(LINE_COMMENT);
}

File: src/main/kotlin/raft/war/language/lni/psi/LniTokenSets.kt
package raft.war.language.lni.psi

import com.intellij.psi.tree.TokenSet
import raft.war.language.lni.psi.LniTypes.LINE_COMMENT

interface LniTokenSets {
    companion object {
        val COMMENTS: TokenSet = TokenSet.create(LINE_COMMENT)
    }
}

